src/app/common/schemas.py

```diff
@@ -125,28 +185,112 @@ def load_from_fitz_rect(rect: fitz.Rect) -> "BoundingBox":
 
 
 class Coordinates(BaseModel):
-    """Coordinates schema."""
+    """Coordinates schema for representing geographical data points.
 
-    east: float = Field(..., example=1.0)
-    north: float = Field(..., example=2.0)
-    projection: str = Field(..., example="LV95")
+    This schema defines the format for specifying location data using east/north coordinates
+    along with the projection system used for geo-referencing.
+    """
+
+    east: float = Field(
+        ...,
+        description="""
+            Easting coordinate, representing the horizontal position of the point. The value should be in the 
+            units of the specified projection system.
+        """,
+        example=1.0,
+    )
+    north: float = Field(
+        ...,
+        description="""
+            Northing coordinate, representing the vertical position of the point. The value should be in the 
+            units of the specified projection system.
+        """,
+        example=2.0,
+    )
+    projection: str = Field(
+        ...,
+        description="""
+            Projection system used to reference the coordinates. This defines the coordinate reference system, 
+            such as 'LV95' for Swiss coordinate systems.
+        """,
+        example="LV95",
+    )
 
 
 class ExtractDataRequest(ABC, BaseModel):
-    """Request schema for the extract_data endpoint.
+    """Request schema for the `extract_data` endpoint.
+
+    **Coordinate Systems:**
+    - **PNG coordinates:** Pixels are measured from the top-left corner (0, 0), where x increases rightward
+    and y downward.
+    - **PDF coordinates:** Also measured from the top-left corner (0, 0), though any transformations between
+    PDF and PNG coordinates are managed internally by the `BoundingBox.rescale` method.
```

Comment:
Is it necessary to explain PDF coordinates here? I don't immediately see where the API user needs to know about PDF coordinates, as they are not used in the API anywhere.

Reply:
I agree with you. That will only confuse people. I also removed the PDF reference a bit later in the text.